manage/users.php

```diff
@@ -396,7 +419,8 @@ function csrf_validate(&$mydata, $name) {
     <td><a href="https://people.php.net/?username=<?php echo hsc($userdata['username']) ?>"><?php echo hsc($userdata['username']) ?></a></td>
     <td><?php echo hsc($userdata['name']);?></td>
 <?php if (!$unapproved) { ?>
-    <td colspan="2"><?php echo hsc($userdata['email']);?></td>
+    <td><?php echo hsc($userdata['email']);?></td>
+    <td><a href="https://github.com/<?php echo hsc($userdata['github']) ?>"><?php echo hsc($userdata['github']);?></a></td>
```

Comment:
I haven't checked it, does `hsc()` use `ENT_QUOTES`? Otherwise the quotes for the attribute won't be escaped properly.

Reply:
Yes, it uses.

```
function hsc($str){ return htmlspecialchars((string)$str,ENT_QUOTES,'UTF-8'); }
```